lib/myfinance/resources/collection.rb

```diff
@@ -0,0 +1,57 @@
+module Myfinance
+  module Resources
+    #
+    # A wrapper to Myfinance collection returns from API. This wrapper represents a collection and it's responsible for processing pagination information as well.
+    #
+    class Collection < Base
+      PAGE_REGEX = /page=(\d+)/
+
+      attr_reader :response, :collection, :headers
+
+      def initialize(response)
+        @response = response
+        @collection = []
+        @headers = response.headers['Link'].split(',') rescue []
```

Comment:
Avoid using `rescue` in its modifier form.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.